### PR TITLE
generalhw fixes

### PR DESCRIPTION
**Pull request by ggardet**
Tested locally as o3 does not support it yet.

### CODE REVIEW

**Comment by ggardet** [comment]
`get_sol_over_SSH.sh` requires https://github.com/os-autoinst/os-autoinst/pull/1353

File: data/generalhw_scripts/flash_usb_otg.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@ destination=$2
 # Size to resize (optionnal, default to 24G)
 size=${3:-24G}
 # user
-username=$(whoami)
+username=root # $(whoami)
```
Comment (Martchus) [inline data/generalhw_scripts/flash_usb_otg.sh]: Can you add the reason for this change in the commit message body?

Comment (ggardet) [reply data/generalhw_scripts/flash_usb_otg.sh]: Currently, it is easier to setup the whole thing, if `root` is used everywhere in the flash script. 
I will fix this later to handle a non-root user, but for now I use `root` user to move forward on other topics. So, no real good reason. ;)

Comment (Martchus) [reply data/generalhw_scripts/flash_usb_otg.sh]: Then at least mention this. Otherwise people are wondering about the out commented code.

**Comment by Martchus** [comment]
The commit message is still not starting with a verb in imperative. I don't want you to update the message for the shake of consistent grammar but because otherwise it really lacks the essence of what this change is about. I assume it changes whether ttys are used for the current backend. So what about: `Prevent using ttys when using generalhw backend without VNC`

**Comment by ggardet** [comment]
Updated accoringly.

File: data/generalhw_scripts/get_sol_over_SSH.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```
Comment (okurz) [inline data/generalhw_scripts/get_sol_over_SSH.sh]: can you use `#!/bin/sh -e` by default to rely on POSIX sh and abort on errors. Use `#!/bin/bash -e` when you need actual bash features

Comment (ggardet) [reply data/generalhw_scripts/get_sol_over_SSH.sh]: Fixed.

**Comment by ggardet** [comment]
It should be ready now.

**Pull request merged**